openpype/hosts/maya/api/plugin.py

```diff
@@ -263,8 +271,19 @@ def update(self, container, representation):
                 "{}:*".format(namespace), type="AlembicNode"
             )
             if alembic_nodes:
-                for attr, value in alembic_data.items():
-                    cmds.setAttr("{}.{}".format(alembic_nodes[0], attr), value)
+                for attr, data in alembic_data.items():
+                    node_attr = "{}.{}".format(alembic_nodes[0], attr)
+                    if data["input"]:
+                        cmds.connectAttr(
+                            data["input"], node_attr, force=True
+                        )
```

Comment:
Shouldn't this make sure that the connection to be made doesn't already exist? (is a warning issues when e.g. `time` was previously connected and you'd update (the update reconnects time by default) and thus this tries to connect time again even though already connected. Does that error or log a warning?

Reply:
note from testing side...it didnt reconnect ```time``` by default after updating the asset so no chance for any warning or error. If I am correct.